config/auth0.php

```diff
@@ -6,30 +6,33 @@
  * Please review available configuration options here:
  * https://github.com/auth0/auth0-PHP#configuration-options.
  */
+$isKeycloakEnabled = env('IS_KEYCLOAK_ENABLED', false);
+
 return [
```

Comment:
Maybe it is more readable to start with the if statement and "early return" an array if Keycloak login is enabled. When Keycloak is not enabled the existing Auth0 config is returned. This will also make the cleanup later more straight forward.

Reply:
I see some tenants need to be configured always. So you could also create the login array first and merge it with the tenants part.

Reply:
I changed the approach, i now split up the variables and change everything in the service container layer.
I also did test and all auth0 client creation still works.